### PR TITLE
Update Scala version to 2.12.13 & bump scalameta

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

**Comment by bjaglin** [comment]
you should be able to fix Mia by updating https://github.com/scalacenter/scalafix/commit/cc98b85a4671f983bb68bdb3cbc630ca5c130169#diff-cf70a614681abb9e33115a49e0b4a52865bdfcdb24c4c0df4af6546d92f88f5f

**Comment by tgodzik** [comment]
> you should be able to fix MiMa by updating [cc98b85#diff-cf70a614681abb9e33115a49e0b4a52865bdfcdb24c4c0df4af6546d92f88f5f](https://github.com/scalacenter/scalafix/commit/cc98b85a4671f983bb68bdb3cbc630ca5c130169#diff-cf70a614681abb9e33115a49e0b4a52865bdfcdb24c4c0df4af6546d92f88f5f)

Thanks! Though it seems it's still failing:
```
sbt.librarymanagement.ResolveException: unresolved dependency: ch.epfl.scala#scalafix-reflect_2.13.3;0.9.24: not found
```

that should be avialable really, no?

File: project/ScalafixBuild.scala [hunk 1]
```diff
@@ -266,7 +266,10 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     )
   )
 
-  private val PreviousScalaVersion = Map[String, String]()
+  private val PreviousScalaVersion = Map(
+    "2.12.13" -> "2.12.12",
+    "2.13.4" -> "2.13.3"
```
Comment (bjaglin) [inline project/ScalafixBuild.scala:271]: you didn't actually bump 2.13 in this PR, so you should remove this line as it tries to get the previous artifact with 2.13.3 while it was already built for 2.13.4

Comment (tgodzik) [reply project/ScalafixBuild.scala]: Ach, my bad! Somehow, I thought it wan't yet updated to 2.13.4.

**Review by bjaglin — APPROVED** [review]

File: bin/test-release.sh [hunk 2]
```diff
@@ -9,12 +9,12 @@ cs resolve \
   ch.epfl.scala:scalafix-core_2.12:$version  \
   ch.epfl.scala:scalafix-core_2.13:$version  \
   ch.epfl.scala:scalafix-reflect_2.11.12:$version  \
-  ch.epfl.scala:scalafix-reflect_2.12.12:$version  \
-  ch.epfl.scala:scalafix-reflect_2.13.3:$version  \
+  ch.epfl.scala:scalafix-reflect_2.12.13:$version  \
+  ch.epfl.scala:scalafix-reflect_2.13.4:$version  \
```
Comment (bjaglin) [inline bin/test-release.sh:13]: thanks!

**Pull request merged**